pkg/fileutils/fileutils.go

```diff
@@ -52,7 +52,7 @@ func CleanPatterns(patterns []string) ([]string, [][]string, bool, error) {
 		if exclusion(pattern) {
 			pattern = pattern[1:]
 		}
-		patternDirs = append(patternDirs, strings.Split(pattern, "/"))
+		patternDirs = append(patternDirs, strings.Split(pattern, string(os.PathSeparator)))
```

Comment:
@cpuguy83 No, it's not a filepath.Split - that just gets a single dir and file. This is doing a split of all components.